### PR TITLE
V1 updates for `brainreg` changes

### PR DESCRIPTION
**Pull request by willGraham01**
Closes https://github.com/brainglobe/BrainGlobe/issues/39

Logs the changes that are being made to the `brainreg`, `brainreg-napari`, and `brainreg-segment` packages to the website.

Also starts on the `brainglobe` version 1 changelog page.

These pages are currently not linked to from anywhere on the website, although they are none-the-less accessible.

### CODE REVIEW

**Comment by willGraham01** [comment]
> Maybe we should create a development section

We do already have a `developers` folder (and presumably page) so maybe it should go there? Although we probably want a front-facing "version 1 is coming" page too

**Comment by alessandrofelder** [comment]
Yea maybe we need a "Releases" section for changelogs? And a "Blogs" session somewhere close to the front for announcements such as this, that links to the relevant changelog?

**Comment by adamltyson** [comment]
> We do already have a developers folder (and presumably page) so maybe it should go there?
We have a developers folder, but it's (currently) linked within documentation, so we should probably make that accessible somewhere else. 

> Yea maybe we need a "Releases" section for changelogs?
Ideally

>  And a "Blogs" session somewhere close to the front for announcements such as this, that links to the relevant changelog?

Yup, pydata-sphinx has a blogging extension we can use (we use it for https://neuroinformatics.dev/)

> I think this is separate from the general developer docs - it's an announcement that we mostly need users and developers of other packages (also a kind of user I guess) to be aware of?

I think the blog can announce to everyone, and then this can link to specific docs for developers and/or users

FYI rather than changing the home page, we can add [an announcement](https://pydata-sphinx-theme.readthedocs.io/en/stable/user_guide/announcements.html) that links to the blog.

**Comment by willGraham01** [comment]
So end-goal would be a structure like the following (within `docs/source`):

```
blogs/
  version_1_announcement.md                      # Main "this is happening" announcement
  brainreg_updated_to_version_1_annoucement.md   # Added as incremental packages are updated
  cellfinder_updated_to_version_1_annoucement.md # etc
  brainglobe_vesion_1_released.md                # :partying_face: 

index.md        # Gets an announcement directing to version_1_annoucement.md

releases/
  v1/
    full_changelog.md          # High level overview, TL;DR style
    brainreg.md                # Detailed listing of what has changed name, 
    cellfinder.md              # where functionality has been moved to, etc, on a
                               # package-by-package basis
documentation/
  # Individual pages to be updated to version 1 compatible information as appropriate.
```

On my to-do list today / Tuesday is the Blog announcement so I can start a separate PR for that, then rebase this on top.

**Comment by adamltyson** [comment]
> So end-goal would be a structure like the following (within `docs/source`):
> 
> ```
> blogs/
>   version_1_announcement.md                      # Main "this is happening" announcement
>   brainreg_updated_to_version_1_annoucement.md   # Added as incremental packages are updated
>   cellfinder_updated_to_version_1_annoucement.md # etc
>   brainglobe_vesion_1_released.md                # :partying_face: 
> 
> index.md        # Gets an announcement directing to version_1_annoucement.md
> 
> releases/
>   v1/
>     full_changelog.md          # High level overview, TL;DR style
>     brainreg.md                # Detailed listing of what has changed name, 
>     cellfinder.md              # where functionality has been moved to, etc, on a
>                                # package-by-package basis
> documentation/
>   # Individual pages to be updated to version 1 compatible information as appropriate.
> ```
> 
> On my to-do list today / Tuesday is the Blog announcement so I can start a separate PR for that, then rebase this on top.

100% thanks!

File: docs/source/index.md [hunk 1]
```diff
@@ -39,6 +39,14 @@ How do I get started with these tools?
 Find out more about these tools.
 :::
 
+:::{grid-item-card} {fas}`tags;sd-text-primary` Releases
+:img-bottom: _static/atlases.png
```
Comment (willGraham01) [inline docs/source/index.md:43]: Do we have a good image to use for the "releases" card? Alternatively, we don't need a card but I feel like it warrants one.

Similarly we don't have a "blogs" card either, if we wanted to include that.

Comment (adamltyson) [reply docs/source/index.md:43]: I don't think we should have a "releases" card. The original idea of the cards was to highlight the main things that new users may want to find out about. I think the majority of users don't care about releases and changelogs.

**Comment by adamltyson** [comment]
Sorry @willGraham01 I didn't fully understand the structure you outlined earlier. I don't think releases should be in the top bar or in a card. I think it's just not important enough for most users for it to be presented so prominently. 

As suggested by @niksirbi, what about a "community" section to bring some different parts together? I'll update this PR and you can take a look. 

As an aside, can the blog list posts in reverse chronological order instead?

**Comment by willGraham01** [comment]
> what about a "community" section to bring some different parts together? I'll update this PR and you can take a look.

No problem, shout when you've pushed things up. (I've just removed the releases card on the main page).

> As an aside, can the blog list posts in reverse chronological order instead?

I assume so, I also thought that was the default. Let me investigate

**Review by alessandrofelder — APPROVED** [review]
Great, thanks @willGraham01 and @adamltyson . Looks good and ready for the big merge!

**Pull request merged**